_ml-commons-plugin/api/model-apis/batch-predict.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: default
+title:  Batch inference
+parent: Model APIs
+grand_parent: ML Commons APIs
+nav_order: 20
+---
+
+# Batch inference
+
+ML Commons can predict large datasets in an offline asynchronous mode with your remote model deployed in external model servers. To use the Batch_Predict API, the `model_id` for a remote model is required. This new API is released as an experimental feature in the OpenSearch version 2.16, and only SageMaker, Cohere, and OpenAI are verified as the external servers that support this features.
```

Comment:
1. Don't we need a `experimental` tag somewhere?
2. Are you planning to add examples for `SageMaker` & `Cohere` too?